BaSP-a2022/ayom-a-trackgenix-app
# BaSP-TG-131: Delete account employee

- Add functionality to delete form button
- Show confirmation and message buttons
- Once deleted, redirect to login

How to test?
- login as an Employee
- click on the delete account button
- test the correct functionality
- watch the result on mongoDB (isDeleted field should be true)
- after testing is done revert the changes on mongoDB
